### PR TITLE
Heartbeat: built successfully

### PR DESCRIPTION
**Pull request by vmoravec**
- I must have commented out two lines in one file; to me it looked like it's bug in the original code in `src/include/heartbeat/log_conf.ycp`

### CODE REVIEW

**Comment by vmoravec** [comment]
I don't know how to test this module, any hints welcome.

File: patches/heartbeat.patch [hunk 1]
```diff
@@ -22,6 +56,40 @@ index 4d29b98..c049388 100644
...
+ {
+     return true;
+@@ -93,7 +92,9 @@ term timeouts_conf_getDialog()
+ 
+ any ConfigureTimeoutsDialog () {
+ 
+-    timeouts_conf_Read();
++   // FIXME not defined in this file; 
++   // it's present in the timeouts_conf.ycp file but that is not included
++   //timeouts_conf_Read();
```
Comment (jreidinger) [inline patches/heartbeat.patch:77]: that is not problem, as it is another include, so it expect that someone who include this file also include timeouts_conf.ycp and include it before this file ( so it is not good way how to do it, but possible ).

**Comment by dmajda** [comment]
Please improve the commit messages so that it is possible to actually tell what is being done in the commit without looking at the code. An important part of that is mentioning what module is the commit about (you can't assume that one will look at the commits only in the context af a pull request).

In Studio, we have a [guide about writing commit messages](https://github.com/SUSE/studio/blob/master/docs/README.commits). I think it would be a good idea to read it and use it.

File: patches/heartbeat.patch (hunk 1 above)
Comment (vmoravec) [inline patches/heartbeat.patch:77]: Agree, but the weird thing is that this method returns some data which is here thrown away. The other function timeouts_conf_Write is even missing the params, it looks like not finished.

File: patches/heartbeat.patch [hunk 2]
```diff
@@ -22,6 +56,40 @@ index 4d29b98..c049388 100644
...
  include "heartbeat/common.ycp";
+diff --git a/src/include/heartbeat/log_conf.ycp b/src/include/heartbeat/log_conf.ycp
+index 1302d3a..8b2d23c 100644
+--- a/src/include/heartbeat/log_conf.ycp
++++ b/src/include/heartbeat/log_conf.ycp
+@@ -59,7 +59,6 @@ import "SCR";
+ include "heartbeat/helps.ycp";
+ include "heartbeat/common.ycp";
+ 
+-
```
Comment (lslezak) [inline patches/heartbeat.patch]: This chunk just removes one empty line, remove it from the patch.

**Comment by vmoravec** [comment]
Commit messages and patch file updated

**Pull request merged**